src/catalog/database_catalog.cpp

```diff
@@ -1661,6 +1712,41 @@ bool DatabaseCatalog::CreateTableEntry(transaction::TransactionContext *const tx
   return true;
 }
 
+std::vector<std::pair<uint32_t, postgres::ClassKind>> DatabaseCatalog::GetNamespaceObjectOids(
```

Comment:
nit: This is labeled "object" OIDs but it is only "class" OIDs and does not include querying `pg_type`.

I know we haven't implemented that really, but where are we thinking that would get hooked in?  Here or in the main namespace function?